flatpak/manifest_tool/cli.py

```
@@ -771,6 +814,59 @@ def _run_find_flutter_sdk(namespace: argparse.Namespace) -> int:
+        flatpak_flutter_timeout=_env_optional_int("FLATPAK_FLUTTER_TIMEOUT"),
+        extra_env=extra_env,
+        test_build=_env_bool("TEST_BUILD", False),
+        flathub_dir=(
+            namespace.flathub_dir.resolve() if namespace.flathub_dir else None
+        ),
+    )
+
+    try:
+        prepare_flathub(options)
+    except PrepareFlathubError as exc:
+        logger.exception("prepare-flathub failed: %s", exc)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)